### ISSUE TITLE
Loads, but crashes when run

### ISSUE THREAD

**Issue by Nodd** [spyder-ide/spyder.line_profiler#4]
The plugin now loads fine in spyder v3.0, but if a profiling is started it crashes spyder with `Unhandled Python exception`.

**Comment by Nodd** [comment]
The following error is raised:

```
Traceback (most recent call last):
  File "/home/joseph/.spyder-py3/spyplugins/spyder_line_profiler/lineprofiler.py", line 141, in run_lineprofiler
    self.analyze(self.main.editor.get_current_filename())
  File "/home/joseph/.spyder-py3/spyplugins/spyder_line_profiler/lineprofiler.py", line 160, in analyze
    use_colors=self.get_option('use_colors', True))
  File "/home/joseph/.spyder-py3/spyplugins/spyder_line_profiler/widgets/lineprofiler.py", line 195, in analyze
    self.start(wdir, args, pythonpath)
  File "/home/joseph/.spyder-py3/spyplugins/spyder_line_profiler/widgets/lineprofiler.py", line 247, in start
    self.process.setEnvironment(env)
AttributeError: 'QProcess' object has no attribute 'setEnvironment'
```

I suppose it needs to be updated for Qt5

**Comment by Nodd** [comment]
Note: it's fixed in my repo, I'll merge when the new plugin structure in spyder is merged.

**Comment by jitseniesen** [comment]
Fixed in 1e0cf584a
